### DISCUSSION TITLE
Separate files are not generated

### DISCUSSION THREAD

**Discussion by GitEddyLFE** [CarlGao4/Demucs-Gui#36] [Q&A]
I am using demucs 1.0 the latest version, and inside the interface when I select file format in: wav and samplerate in: 24bits, the folder where it has been created appears empty, without any audio file, and apparently only when I leave the default in "flac" - "int16" if it is exported, is there any way to solve this?

**Comment by CarlGao4** [comment]
Can't reproduce. Please attach the log file (you can open the folder from the menu bar). You can remove some sensitive information (like the file path containing your username, but only remove your username in this case) before uploading it.
